frontend/appflowy_flutter/packages/flowy_infra_ui/android/.project

```diff
@@ -22,12 +22,12 @@
 	</natures>
 	<filteredResources>
 		<filter>
-			<id>1626576261667</id>
+			<id>1693395487121</id>
```

Comment:
Please revert the changes related to Android if they are not required.

Reply:
done ✅